dataset-cellsystems-2021/live/feature_defs.json

```diff
@@ -0,0 +1,160 @@
+    "unit": "\u00b5m\u00b2",
+    "key": "area-diffuse",
+    "discrete": false
+  },
+  {
+    "displayName": "Fraction area background",
+    "description": "Fraction of cell area covered by background, one of the five local patterns classified by the machine learning based alpha-actinin-2 pattern classifier.",
+    "tooltip": "Fraction of cell area covered by background, one of the five local patterns classified by the machine learning based alpha-actinin-2 pattern classifier.",
+    "unit": "\u00b5m\u00b2",
+    "key": "area-background",
+    "discrete": false
+  },
+  {
+    "displayName": "Max coefficient of variation",
+    "description": "Maximum coefficient of variation is one of three key Haralick correlation based metrics of global sarcomere structure alignment.",
+    "tooltip": "Maximum coefficient of variation is one of three key Haralick correlation based metrics of global sarcomere structure alignment.",
+    "unit": "",
+    "key": "max-cov",
+    "discrete": false
+  },
+  {
+    "displayName": "Peak height",
+    "description": "Peak height is one of three key Haralick correlation based metrics of global sarcomere structure alignment.",
+    "tooltip": "Peak height is one of three key Haralick correlation based metrics of global sarcomere structure alignment.",
+    "unit": "",
+    "key": "peak-height",
+    "discrete": false
+  },
+  {
+    "displayName": "Peak distance",
+    "description": "Peak distance is one of three key Haralick correlation based metrics of global sarcomere structure alignment.",
+    "tooltip": "Peak distance is one of three key Haralick correlation based metrics of global sarcomere structure alignment.",
+    "unit": "\u00b5m",
+    "key": "peak-distance",
+    "discrete": false
+  },
+  {
+    "displayName": "Alpha-actinin-2 protein density",
+    "description": "Alpha-actinin-2 protein density in 2D segmented cell.",
+    "tooltip": "Alpha-actinin-2 protein density in 2D segmented cell.",
+    "unit": "a.u. x10\u00b6",
+    "key": "protein-density",
+    "discrete": false
+  },
+  {
+    "displayName": "Cell area",
+    "description": "2D segmented cell area.",
+    "tooltip": "2D segmented cell area.",
+    "unit": "\u00b5m\u00b2",
+    "key": "cell-area",
+    "discrete": false
+  },
+  {
+    "displayName": "Cell aspect ratio",
+    "description": "2D segmented cell aspect ratio.",
+    "tooltip": "2D segmented cell aspect ratio.",
+    "unit": "",
+    "key": "cell-aspect",
+    "discrete": false
+  },
+  {
+    "displayName": "Experiment",
+    "description": "Differentiation experiment encompassing all plates and wells setup and replated on a given day.",
+    "tooltip": "Differentiation experiment encompassing all plates and wells setup and replated on a given day.",
+    "unit": "",
+    "key": "diff-exp",
+    "discrete": true
```

Comment:
if a feature is discrete it needs to have the `options` list

Reply:
can the options list just have a name?
 ```
   "options": {
        "1": {
            "name": "Exp1"
        },
        "2": {
            "name": "Exp2"
        },
        "3": {
            "name": "Exp3"
        }
```

Reply:
I think currently it needs a color. If you're having trouble picking them, this is a good site: https://colorbrewer2.org/#type=sequential&scheme=BuGn&n=3